packages/frontend/core/src/pages/workspace/detail-page/editor-sidebar/extensions/journal.css.ts

```diff
@@ -151,4 +158,44 @@ export const pageItemLabel = style({
   fontSize: 'var(--affine-font-size-sm)',
   color: 'var(--affine-text-primary-color)',
   textAlign: 'left',
+  selectors: {
+    '[aria-selected="true"] &': {
+      // TODO: wait for design
+      color: 'var(--affine-primary-color)',
+    },
+  },
+});
+
+// conflict
+export const journalConflictBlock = style({
+  padding: '0 16px 16px 16px',
+});
+export const journalConflictWrapper = style({
+  display: 'grid',
+  gridTemplateColumns: 'repeat(auto-fill, minmax(170px, 1fr))',
+  rowGap: 4,
+  columnGap: 8,
 });
+export const journalConflictMoreTrigger = style([
+  interactive,
+  {
+    color: 'var(--affine-text-secondary-color)',
+    height: 30,
+    borderRadius: 4,
+    padding: '0px 8px',
+    fontSize: 'var(--affine-font-size-sm)',
+    display: 'flex',
+    alignItems: 'center',
+  },
+]);
+
+// override date-picker's active day when is not journal
+globalStyle(
```

Comment:
Use selectors? I think this kind of globalStyle usage is mislead.

Reply:
The date cell in global `AFFiNEDatePicker` is not customizable, but we need to override style only for journal. I'm not sure what u mean 'use selectors'?

Reply:
is it get invalid selectors error ？

Reply:
This syntax is not allowed by `@vanilla-extract/css`, and will cause error: `Style selectors must target the '&' character`

> https://vanilla-extract.style/documentation/styling/#complex-selectors
> To improve maintainability, each style block can only target a single element. To enforce this, all selectors must target the `&` character which is a reference to the current element.